scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -0,0 +1,304 @@
+#
+# Workflow a single pair of case-control samples.
+#
+# Notes:
+#
+# - file names will use the entity ID specified, but inside the file, the bam SM tag will typically be used.
+#
+# - THIS SCRIPT SHOULD BE CONSIDERED OF "BETA" QUALITY
+#
+# - For jni_lib values, see README.  For example, Ubuntu: /usr/lib/jni
+#
+###########
+
+workflow case_gatk_acnv_workflow {
+    String wf_entity_id_tumor
+    String wf_entity_id_normal
+    File target_bed
+    File ref_fasta
+    File ref_fasta_dict
+    File ref_fasta_fai
+    File common_snp_list
+    File tumor_bam
+    File tumor_bam_idx
+    File normal_bam
+    File normal_bam_idx
+    File jar_file
+    File PoN
+    String is_disable_reference_validation
+    File jni_lib    
+
+  call PadTargets {
+    input:
+        target_bed=target_bed,
+        jar_file=jar_file,
+        mem=1
+  }
+
+  call CalculateTargetCoverage as TumorCalculateTargetCoverage {
+    input:
+        entity_id=wf_entity_id_tumor,
+        padded_target_bed=PadTargets.padded_target_bed,
+        input_bam=tumor_bam,
+        ref_fasta=ref_fasta,
+        ref_fasta_fai=ref_fasta_fai,
+        ref_fasta_dict=ref_fasta_dict,
+        jar_file=jar_file,
+        disable_reference_validation=is_disable_reference_validation,
```

Comment:
why the different names (i.e. `is_`) on the inner scope vs outer scope variable names?